src/proto/streams/counts.rs

```diff
@@ -46,6 +46,10 @@ impl Counts {
         self.peer
     }
 
+    pub fn has_streams(&self) -> bool {
+        self.num_recv_streams != 0 || self.num_reset_streams != 0
```

Comment:
I think it was num_reset_streams that was supposed to be ignored? num_send_streams is important. Is that correct @seanmonstar?

Reply:
Yes, it is `num_reset_streams` that isn't needed.

Reply:
Oh crap, my bad. Updated.